es-core/src/Settings.cpp

```diff
@@ -113,8 +113,7 @@ void Settings::setDefaults()
 	mBoolMap["CaptionsCompatibility"] = true;
 	// Audio out device for Video playback using OMX player.
 	mStringMap["OMXAudioDev"] = "both";
-	mStringMap["CollectionSystemsAuto"] = "";
-	mStringMap["CollectionSystemsCustom"] = "";
+	mStringMap["EnabledCollectionSystems"] = "";
```

Comment:
Thanks.

Just a small note, for the sake of compatibility: any chance you'd consider building a small "importer" (i.e. check for the existence of the previous settings on init, and initialize this string then)?

As you're refactoring these settings and their names, when this is launched end users will lose their settings and ask where did the collections go. I'm unsure if there's an easy way to do that that doesn't really complain about the missing settings string, but worth considering.

Also, for display purposes (and I'm not sure if that was intentional) we did display them separately - the automatic ones were always at the end of the systems list, acting as quick index/reference.

Reply:
Do you think a conversion script is really worthwhile? In my (professional) experience that way lies nightmares with conversion levels and skipped/missed conversions when upgrading from old releases with multiple level differnces </shudder>. 

But I see the user need, as well as our need to minimize required support after introducing new features. 
Maybe we could just retain the old functionality, append the two lists/strings upon loading and store them in the new key, followed by removing the old keys.

When I get time, I will brew something up for this, let me get back to you on this.

Reply:
I mean, truth be told it's a one off, so it likely isn't a necessity. People will find that their collections aren't visible, they'll go to the menu and enable them. Not a big issue. Feel free to disregard.

What I had in mind was really what you described, to retain the old functionality, append the two lists/strings upon loading and store them in the new key, followed by removing the old keys.

But, in reality, that just adds to the code we'll need to maintain in the future, and then few will even remember why that is there.

I vote to ignore my initial reaction, really. I'm always wary of user experience breaking, hence my knee-jerk reaction, but I'm certainly blowing things out of proportion here. :)